eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -63,19 +76,30 @@ jobs:
       - template: /eng/pipelines/templates/steps/install-dotnet.yml
         parameters:
           NuGetCacheKey: Test
+      - pwsh: | 
+          if ("${{ parameters.ProjectListOverrideFilePropertyName }}" -ne '') {
+            Write-Host "##vso[task.setvariable variable=ProjectListOverrideFileName]$(${{ parameters.ProjectListOverrideFilePropertyName }})"
+            Write-Host "##vso[task.setvariable variable=CodeCoverageFilePattern]$(Build.SourcesDirectory)\sdk\**\coverage.cobertura.xml"
```

Comment:
Per your Teams message, this is probably fine. I'm pretty certain the only reason I limited it to the service directory was to avoid more exhaustive directory scans. Since this would only run during the CI, it would be much less noticeable.